src/test/java/com/wikia/webdriver/testcases/adstests/TestAdTypeMobile.java

```diff
@@ -1,119 +1,94 @@
 package com.wikia.webdriver.testcases.adstests;
 
-import com.wikia.webdriver.common.dataprovider.ads.AdTypeDataProvider;
+import com.wikia.webdriver.common.contentpatterns.AdsContent;
+import com.wikia.webdriver.common.core.annotations.InBrowser;
+import com.wikia.webdriver.common.core.drivers.Browser;
+import com.wikia.webdriver.common.core.helpers.Emulator;
+import com.wikia.webdriver.common.core.url.Page;
 import com.wikia.webdriver.common.templates.mobile.MobileTestTemplate;
 import com.wikia.webdriver.pageobjectsfactory.pageobject.adsbase.mobile.MobileAdsBaseObject;
-
 import org.openqa.selenium.By;
 import org.testng.annotations.Test;
 
 /**
  * https://www.google.com/dfp/5441#delivery/OrderDetail/orderId=245575332
  */
+@InBrowser(
+  browser = Browser.CHROME,
+  emulator = Emulator.GOOGLE_NEXUS_5
+)
+@Test(groups = "AdTypeMercury")
 public class TestAdTypeMobile extends MobileTestTemplate {
-
-  private static final String SRC = "mobile";
-
-  @Test(
-      groups = "AdTypeMercury",
-      dataProviderClass = AdTypeDataProvider.class,
-      dataProvider = "asyncSuccessWithAd"
-  )
-  public void adsAdTypeAsyncSuccessWithAd(String wikiName, String article, String adUnit,
-                                          String slotName, String imgUrl) {
-    String testedPage = urlBuilder.getUrlForPath(wikiName, article);
-    MobileAdsBaseObject ads = new MobileAdsBaseObject(driver, testedPage);
-    ads.waitForSlot(slotName);
-    ads.verifyGptIframe(adUnit, slotName, SRC);
-    ads.verifyImgAdLoadedInSlot(slotName, imgUrl);
+  private static final String DFP_IMAGE_URL =
+      "googlesyndication.com/pagead/imgad?id=CICAgKCNj62dEhCsAhj6ASgBMgjBw3U0lR5Thg";
+  
+  @Test
+  public void adsAdTypeAsyncSuccessWithAd() {
+    // https://www.google.com/dfp/5441#delivery/LineItemDetail/lineItemId=260851332&orderId=245575332
+    Page page = new Page("project43", "SyntheticTests/AdType/Async/Success");
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "project43" 7 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)